Modules/Customer/Http/Services/CustomerService.php

```diff
@@ -199,24 +203,23 @@ public function getCustomers(Request $request): array
     /**
      * Remove the specified customer from storage.
      *
-     * @param string $customerKey
+     * @param array $requestData
      * @return array
      */
-    public function delete(string $customerKey): array
+    public function delete(array $requestData): array
```

Comment:
@arijit1111 The list of customer IDs should be passed through the body of the request instead of URL path